evm_trace/display.py

```diff
@@ -39,6 +39,8 @@ def title(self) -> str:
             address = to_checksum_address(address_hex_str) if address_hex_str else None
         except ImportError:
             # Ignore checksumming if user does not have eth-hash backend installed.
+        except ValueError:
+            # a contract may attempt to call into an address that can't be normalized
             address = address_hex_str  # type: ignore
```

Comment:
I'm not as familiar with `evm_trace` but I imagine both `ImportError` and `ValueError` would need to lead to the `address = address_hex_str` path? In which case these current changes have a syntax error and would need to be:

```python
except ImportError:
    # Ignore checksumming if user does not have eth-hash backend installed.
    address = address_hex_str
except ValueError:
    # A contract may attempt to call into an address that can't be normalized.
    address = address_hex_str
```

I kept them both as separate `except`s since they each have an associated comment

Reply:
Updated it!